pyproject.toml

```diff
@@ -33,6 +33,9 @@ dependencies = [
 ]
 dynamic = ["version"]
 
+[tool.setuptools_scm]
+version_file = "src/cantools/version.py"
```

Comment:
Can you try to remove this and instead use this?

```python3
from importlib.metadata import version, PackageNotFoundError

try:
    __version__ = version("package-name")
except PackageNotFoundError:
    # package is not installed
    pass
```

From [here](https://setuptools-scm.readthedocs.io/en/latest/usage/#version-at-runtime)